<launch>
    <rosparam command="load" file="$(find cam2lidar)/config/geometric.yaml"/>
    <node pkg="cam2lidar" name="geometric_calibration" type="geometric_calibration.py" output="screen">
        <!-- <param name="image_topic" value="/usb_cam/image_raw"/> -->
        <param name="image_topic" value="/camera/color/image_raw"/>
        <param name="lidar_topic" value="/velodyne_points"/>
        <param name="distance_threshold" value="15"/>
        <param name="consequent_frames" value="15"/>
        <param name="debug" value="False"/>
    </node>
    <node pkg="cam2lidar" name="user_interface" type="user_interface.py" output="screen">
        <param name="subscriber_name" value="/geometric_visualization"/>
        <!-- <param name="camera_info_topic" value="/usb_cam/camera_info"/> -->
        <param name="camera_info_topic" value="/camera/color/camera_info"/>
    </node>
</launch>